src/frequenz/sdk/core/timeseries.py

```diff
@@ -0,0 +1,71 @@
+"""
+Common model definitions used throughout the SDK.
+
+Copyright
+Copyright © 2022 Frequenz Energy-as-a-Service GmbH
+
+License
+MIT
+"""
+from __future__ import annotations
+
+import datetime
+from dataclasses import dataclass
+
+
+@dataclass(frozen=True)
+class TimeSeriesId:
```

Comment:
May I leave it as it is until it is clear how it will look on the `DataSourcingActor`'s end? Then I'll update it.

Reply:
So, I changed `TimeSeriesId` to be just an alias for `str` and I kept the previous class as `ComponentMetricChannelName`, but I don't know if it will be needed.